tests/system_tests/git/create_repo.sh

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env bash
```

Comment:
Scripts for public use should not require bash. Always use `/bin/sh` as this is guaranteed to exist on all Posix systems. The fact that this is often a link to Bash on Linux systems is not our fault.

Reply:
I thought you would say that :)